omen2/types.py

```diff
@@ -6,23 +6,54 @@
 
 def any_type(arg):
     """Pass-through converter."""
+    # return value as python interpreted
+    return eval(arg)  # pylint: disable=eval-used
+
+
+any_type.__name__ = "Any"
+
+
+def bool_type(arg):
+    """Convert sql string to bool, this function must have the __name__ 'bool'"""
+    arg = arg.lower()
+    if arg == "true":
+        return True
+    elif arg == "false":
+        return False
+    else:
+        raise ValueError(f"Invalid boolean value: {arg!r}")
+
+
+bool_type.__name__ = "bool"
+
+
+def string_type(arg):
+    """Convert sql string to str, this function must have the __name__ 'str'"""
+    assert arg[0] == "'"
```

Comment:
Final nit: what do you think about having this accept double quotes? [SQLite supports them](https://www.sqlite.org/quirks.html#double_quoted_string_literals_are_accepted) but it's non-standard. We do use them in a few places internally, though.

Reply:
ok